kratos/geometries/triangle_3d_3.h

```diff
@@ -623,14 +636,15 @@ template<class TPointType> class Triangle3D3
      *
      * @return The Inradius to Circumradius Quality metric.
      */
-    virtual double AreaToEdgeLengthSquareRatio() const {
-      constexpr double normFactor = 1.0;
+    virtual double AreaToEdgeLengthSquareRatio() const 
+    {
+        constexpr double normFactor = 1.0;
 
-      double a = MathUtils<double>::Norm3(this->GetPoint(0)-this->GetPoint(1));
-      double b = MathUtils<double>::Norm3(this->GetPoint(1)-this->GetPoint(2));
-      double c = MathUtils<double>::Norm3(this->GetPoint(2)-this->GetPoint(0));
+        const double a = MathUtils<double>::Norm3(this->GetPoint(0)-this->GetPoint(1));
+        const double b = MathUtils<double>::Norm3(this->GetPoint(1)-this->GetPoint(2));
+        const double c = MathUtils<double>::Norm3(this->GetPoint(2)-this->GetPoint(0));
 
-      return normFactor * Area() / std::pow(a+b+c, 2);
+        return normFactor * Area() / std::pow(a+b+c, 2);

```

Comment:
std::pow(x, 2) might be replaced by x*x. Who knows if the optimizer is doing it already... @loumalouomega